integrations/docker/images/stage-2/chip-build-tizen/tizen-sdk-installer/install.sh

```diff
@@ -232,13 +232,14 @@ function install_tizen_sdk() {
         'capi-system-peripheral-io-*.armv7l.rpm'
         'capi-system-peripheral-io-devel-*.armv7l.rpm'
         'capi-system-resource-1*.armv7l.rpm'
-        'libnsd-dns-sd-*.armv7l.rpm')
+        'libnsd-dns-sd-*.armv7l.rpm'
+        'sensord-*.armv7l.rpm')
     download "$URL" "${PKG_ARR[@]}"
 
     # Tizen Developer Platform Certificate
     URL="http://download.tizen.org/sdk/extensions/Tizen_IoT_Headless/binary/"
     PKG_ARR=(
-        "$TIZEN_VERSION-iot-things-add-ons_*_ubuntu-64.zip")
+        "7.0-iot-things-add-ons_*_ubuntu-64.zip")
```

Comment:
Please add comment why this is not updated

Reply:
Fixed